src/Components/AccessRequestsTable.js

```diff
@@ -303,35 +306,37 @@ const AccessRequestsTable = ({ isInternal }) => {
       <ToolbarContent>
         <ToolbarItem>
           <InputGroup>
-            <Dropdown
-              isOpen={isDropdownOpen}
-              onSelect={(ev) => {
-                setIsDropdownOpen(false);
-                setFilterColumn(ev.target.value);
-                setIsSelectOpen(false);
-                setFiltersDirty(true);
-              }}
-              toggle={
-                <DropdownToggle
-                  onToggle={(isOpen) => setIsDropdownOpen(isOpen)}
-                >
-                  <FilterIcon /> {filterColumn}
-                </DropdownToggle>
-              }
-              // https://marvelapp.com/prototype/257je526/screen/74764732
-              dropdownItems={(isInternal ? [1, 5] : [6])
-                .map((i) => columns[i])
-                .map((colName) => (
-                  // Filterable columns are RequestID, AccountID, and Status
-                  <DropdownItem
-                    key={colName}
-                    value={colName}
-                    component="button"
+            <InputGroupItem>
+              <Dropdown
+                isOpen={isDropdownOpen}
+                onSelect={(ev) => {
+                  setIsDropdownOpen(false);
+                  setFilterColumn(ev.target.value);
+                  setIsSelectOpen(false);
+                  setFiltersDirty(true);
+                }}
+                toggle={
+                  <DropdownToggle
+                    onToggle={(_event, isOpen) => setIsDropdownOpen(isOpen)}
                   >
-                    {capitalize(colName)}
-                  </DropdownItem>
-                ))}
-            />
+                    <FilterIcon /> {filterColumn}
+                  </DropdownToggle>
+                }
+                // https://marvelapp.com/prototype/257je526/screen/74764732
```

Comment:
Do we need this?

Reply:
Probably not, some legacy comment. I'll remove it